Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/SingleCreditCardView.swift

```diff
@@ -0,0 +1,306 @@
+        contentWidthConstraint = contentView.widthAnchor.constraint(equalToConstant: contentViewWidth)
+        contentWidthConstraint.priority = UILayoutPriority(999)
+
+        NSLayoutConstraint.activate([
+            contentView.topAnchor.constraint(equalTo: view.topAnchor),
+            contentView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            contentView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+
+            cardTableView.topAnchor.constraint(equalTo: contentView.topAnchor, constant: UX.distanceBetweenHeaderAndTop),
+            cardTableView.bottomAnchor.constraint(equalTo: buttonsContainerStackView.topAnchor, constant: 0),
+            cardTableView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: UX.tableMargin),
+            cardTableView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -UX.tableMargin),
+
+            buttonsContainerStackView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor, constant: -UX.bottomSpacing),
+            buttonsContainerStackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: UX.tableMargin),
+            buttonsContainerStackView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -UX.tableMargin),
+
+            yesButton.heightAnchor.constraint(equalToConstant: UX.yesButtonHeight),
+            notNowButton.heightAnchor.constraint(equalToConstant: UX.yesButtonHeight),
+            contentWidthConstraint,
+            tableViewHeightConstraint
+        ])
+    }
+
+    // MARK: View Transitions
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+        print("traitCollectionDidChange")
+        setupView()
+        applyTheme()
+    }
+
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+        print("viewWillTransition")
+        let contentViewWidth = UX.contentViewWidth > size.width ? size.width - UX.containerPadding : UX.contentViewWidth
+        contentWidthConstraint.constant = contentViewWidth
+    }
+
+    // MARK: Button Actions
+    @objc
+    private func didTapYes() {
+        self.viewModel.didTapMainButton { error in
+            // error is logged in the view model, but maybe we want to show an error message
+            DispatchQueue.main.async { [weak self] in
+                self?.dismissVC()
+            }
+        }
+    }
+
+    @objc
+    private func didTapNotNow() {
+        dismissVC()
+    }
+
+    @objc
+    private func didTapManageCards() {
+        dismissVC()
+    }
+
+    // MARK: BottomSheet Delegate
+    func willDismiss() {
+    }
+}
+
+// MARK: UITableViewDelegate
+extension SingleCreditCardViewController: UITableViewDelegate, UITableViewDataSource {
```

Comment:
We decided as a team to not use extensions anymore (apart from rare cases): Please add this code to the class itself.